content/applications/finance/fiscal_localizations/colombia.rst

```diff
@@ -37,208 +34,375 @@ localization:
    :alt: Carvajal XML invoice file in Odoo chatter.
 
-The :guilabel:`Electronic Invoice Name` field is now displayed in the :guilabel:`Other Info` tab
+The :guilabel:`Electronic Invoice Name` field is now displayed in the :guilabel:`EDI Documents` tab
 with the name of the XML file. Additionally, the :guilabel:`Electronic Invoice Status` field is
 displayed with the initial value :guilabel:`In progress`.
 
+.. image:: colombia/invoice-sent-status.png
+   :alt: Electronic Invoice EDI Document XML file in the EDI Documents tab.
+
 .. _colombia/invoice-xml:
 
 Reception of legal XML and PDF
-------------------------------
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-The electronic invoice vendor receives the XML file and proceeds to validate the structure and the
-information in it. In the :guilabel:`Action` drop-down menu, select the :guilabel:`Check Carvajal
-Status` button. If everything is correct, the :guilabel:`Electronic Invoice Status` field value
-changes to :guilabel:`Validated`. Then, proceed to generate a legal XML, which includes a digital
-signature and a unique code (CUFE), a PDF invoice that includes a QR code, and the CUFE is also
-generated.
+The electronic invoice vendor (Carvajal) receives the XML file and proceeds to validate its
+structure and information.
 
-After this:
+In the :guilabel:`Action` menu, select the :guilabel:`Check Carvajal Status` button. The
+:guilabel:`Electronic Invoice Status` field value changes to :guilabel:`Validated` if everything
+is correct. Then, proceed to generate a legal XML, which includes a digital signature and a unique
+code (CUFE), a PDF invoice that consists of a QR code, and the previously-generated CUFE code.
 
-- A ZIP containing the legal XML and the PDF is downloaded and displayed in the invoice chatter:
+A ZIP containing the legal electronic invoice in XML format and the invoice in PDF format is
+downloaded and displayed in the invoice chatter:
 
-  .. image:: colombia/zip-invoice-chatter.png
+  .. image:: colombia/invoice-zip.png
      :alt: ZIP file displayed in the invoice chatter in Odoo.
 
-  .. image:: colombia/zip-file-contents.png
-     :alt: XML and PDF contained in invoice ZIP file.
+The electronic invoice status changes to :guilabel:`Accepted`.
+
+Credit notes
+------------
+
+The process for credit notes is the same as for invoices. To create a credit note with reference to
+an invoice, go to :menuselection:`Accounting
+--> Customers --> Invoices`. On the invoice, click :guilabel:`Add Credit Note` and complete the
+following information:
+
+- :guilabel:`Credit Method`: Select the type of credit method.
+
+  - :guilabel:`Partial Refund`: Use this option when it is a partial amount.
+  - :guilabel:`Full Refund`: Use this option if the credit note is for the full amount.
+  - :guilabel:`Full refund and new draft invoice`: Use this option if the credit note is for the
+    total invoice and you need the credit note and auto-create a new draft invoice.
```

Comment:
```suggestion
  - :guilabel:`Full refund and new draft invoice`: Use this option if the credit note is for the
    total invoice and you need the credit note and auto-create a new draft invoice.
```
That sentence is confusing, I'm not sure I understand what it means

Reply:
Updated to include the Odoo description of the field